platforms/core-runtime/build-process-services/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id("gradlebuild.distribution.implementation-java")
+}
+
+description = "Services and types used to setup a build process from a Gradle distribution."
+
+errorprone {
+    disabledChecks.addAll(
+        "StringSplitter",
```

Comment:
IMO its generally better to add these disabled checks inline so that we don't allow developers to add code in the future that violates this check.